src/updatedb.py

```diff
@@ -232,28 +232,28 @@ def uploadResults(data, branch, revision, date):
                         dir = (dir.split('|')[1]).strip()
                         if dir.endswith(filename):
                             dir = dir.split(filename)[0]
-                            failures.append(dir + '/' + filename)
-        # https://treeherder.mozilla.org/api/project/mozilla-central/jobs/1116367/
-        url = "https://treeherder.mozilla.org/api/project/%s/jobs/%s/" % (branch, _id)
-        data1 = fetch_json(url)
+                            failures.add(dir + filename)
```

Comment:
yeah, because we set failures as set() and there is no way to store duplicate data in a set :)